core/src/test/java/edu/ucr/cs/riple/core/tools/CoreTestHelper.java

```diff
@@ -214,10 +200,26 @@ private void compare(Collection<Report> actualOutput) {
     fail(errorMessage.toString());
   }
 
+  /**
+   * Computes the build command for the project template. It includes, changing directory command
+   * from root to project root dir, command to compile the project and the computed paths to config
+   * files which will be passed through gradle command line arguments.
+   *
+   * @return The command to build the project including the command line arguments, this command can
+   *     be executed from any directory,
+   */
+  private String computeBuildCommandWithGradleCLArguments() {
```

Comment:
Is this method still needed? The one that's being called below is `Utility.computeBuildCommandWithGradleCLArguments`, which is identical in its implementation. Is this left over from some refactoring?

Reply:
No it is not needed, thanks for noticing this. c73cc13902755ffe22baa1ac24c0d4eb3854a2a6